Domain/Repositories/ArticleRepository.cs

```diff
@@ -0,0 +1,62 @@
+
+namespace Domain {
+    public class ArticleRepository : Repository<Article>, IArticleRepository
+    {
+        public ArticleRepository() : this("articles") { }
+
+        public ArticleRepository(string name) : base(name) { }
+
+        public Article Create(CreateArticleDto createArticleDto)
+        {
+            Article newArticle = new(
+                Guid.NewGuid().ToString(),
+                createArticleDto.Title,
+                createArticleDto.Link,
+                DateTime.Now.TimeOfDay,
+                DateTime.Now.TimeOfDay
+            );
+
+
+            List<Article?> articles = Read();
+            Write([newArticle, .. articles]);
+
+            return newArticle;
+        }
+
+        public void Delete(string id)
+        {
+            List<Article?> articles = Read();
+            if (articles.Count == 0)
+            {
+                return;
+            }
+
+            int i = articles.FindIndex(article => article?.Id == id);
+
+            if (i == -1)
+            {
+                new NotImplementedException();
+            }
+
+            articles.RemoveAt(i);
```

Comment:
You can use something like this:
articles.RemoveAll(article => article?.Id == id);